src/Sylius/Bundle/ProductBundle/Resources/config/serializer/Model.ProductAttribute.yml

```diff
@@ -6,19 +6,33 @@ Sylius\Component\Product\Model\ProductAttribute:
             expose: true
             type: integer
             xml_attribute: true
+            gorups: [Default, Detailed]
```

Comment:
`groups` 💃 
almost everywhere in this file + in `Model.ProductOption.yml` as well